wrappers/MATLAB/refpropm.m

```diff
@@ -691,4 +706,32 @@
     end
 end
 
+
+function proto = get_header_path()
+    % Systematic lookup method to find possible header file
+    % Start with the BasePath, then look through unix standard places
+    % and end by looking in MATLAB path.
+    if exist('rp_proto64', 'file')

```

Comment:
So basically, if a proto file can be found, it uses that, otherwise, it uses the header?